docs/2.7.9/docs/daml/reference/packages.rst

```diff
@@ -182,6 +182,25 @@ When importing packages this way, the Daml compiler will try to reconstruct the
 
 Because of their flexibility, data-dependencies are a tool that is recommended for performing Daml model upgrades. See the :ref:`upgrade documentation <upgrade-overview>` for more details.
 
+Transitive dependency management
+================================
+
+The Daml compiler identifies each DAR dependency in the project by their ``packageId`` and fully qualified name (Daml project package name and version number).
+
+If you have a Daml project which contains multiple common transitive DAR dependencies, those common transitive dependencies must either:
+
+* Have the exact same contents if they have the same name and version specified in their Daml project's ``daml.yaml`` file, or
+* Have a different version from each others' Daml project's ``daml.yaml`` file if there are differences in the contents.
```

Comment:
```suggestion
* Have different versions of the Daml project's ``daml.yaml`` files if there are differences in the contents.
```

Reply:
This is actually a change in meaning. What we are trying to say is that the `version` entry of the `daml.yaml` file must have a different value. Can you help me clarify that?

Reply:
How about this, just the way you explained it is good!